src/Microsoft.TestPlatform.ObjectModel/Client/Interfaces/ITestPlatform.cs

```diff
@@ -28,15 +28,17 @@ public interface ITestPlatform : IDisposable
         /// </summary>
         /// <param name="requestData">Providing common services and data for Discovery</param>
         /// <param name="discoveryCriteria">Specifies the discovery parameters</param>
+        /// <param name="options">Test platform options.</param>
         /// <returns>DiscoveryRequest object</returns>
-        IDiscoveryRequest CreateDiscoveryRequest(IRequestData requestData, DiscoveryCriteria discoveryCriteria);
+        IDiscoveryRequest CreateDiscoveryRequest(IRequestData requestData, DiscoveryCriteria discoveryCriteria, TestPlatformOptions options);
```

Comment:
>IDiscoveryRequest [](start = 8, length = 17)

this is a public interface. isnt this a breakingchange

Reply:
no this is not, since Tpv2 this Interface is not meant to be implemented by anyone outside vstest.console
If you see in ITestPlatform we have also made the class internal